helix-term/src/application.rs

```diff
@@ -609,12 +603,7 @@ impl Application {
             EditorEvent::LanguageServerMessage((id, call)) => {
                 self.handle_language_server_message(call, id).await;
                 // limit render calls for fast language server messages
-                let last = self.editor.language_servers.incoming.is_empty();
```

Comment:
For context: the `is_empty` here checks that the set of streams is empty rather than that there is no pending message ready to be handled. Detecting that case is a little involved and not very effective for debouncing (see https://github.com/helix-editor/helix/pull/7373#issuecomment-1616716678)